data_science/scripts/asg_utils.py

```diff
@@ -3,12 +3,7 @@
 import sys
 
 
-DATA_SCIENCE_ASG_TAGS = {
-    'name': 'jupyter',
-}
-
-
-def discover_data_science_asg(asg_client):
+def discover_asg(asg_client, tag_name):
```

Comment:
If you're changing the method signature, you should change the docstring to match.